trivial-method-combinations.lisp

```diff
@@ -55,6 +57,10 @@ this library's issue tracker.")
     (mop::long-method-combination
      (values (mop::long-method-combination-arguments method-combination)
              t))
+    #+allegro
+    ((or excl::long-method-combination
+         excl::standard-method-combination)
+     (values (slot-value method-combination 'excl::options) t))
```

Comment:
In this function, we want the `:argument` lambda-list that was provided in `DEFINE-METHOD-COMBINATION`, not the processed options.

Reply:
Couldn't find a way to retrieve the lambda-list. Macro-expanding define-method-combination, the lambda list seems be hidden away in an inner lambda list. Perhaps `c2mop:define-method-combination` could take care of storing the lambda list? That'd be nice.

Reply:
> Perhaps `c2mop:define-method-combination` could take care of storing the lambda list? That'd be nice.

Please ask about that on the `closer-mop` repository. I'll follow suit the moment I get a way of retrieving the args-lambda-list.

Reply:
Come to think of it. This closer-mop solution wouldn't be very robust. It would only work for method combinations defined via this hypothetical c2mop:define-method-combination and not anything else. 😔